modules/rosa-sts-creating-cluster.adoc

```diff
@@ -57,20 +72,68 @@ I: Cluster '<cluster_name>' has been created.
 I: Once the cluster is 'Ready' you will need to add an Identity Provider and define the list of cluster administrators. See `rosa create idp --help` and `rosa create user --help` for more information.
 I: To determine when your cluster is Ready, run `rosa describe cluster <cluster_name>`.
 ----
-** To create a cluster with STS using interactive prompts:
++
+[NOTE]
+====
+You can configure the following default network IP ranges:
+
+* Machine CIDR: 10.0.0.0/16
+* Service CIDR: 172.30.0.0/16
+* Pod CIDR: 10.128.0.0/14
+
+For the CIDR-related `rosa` CLI arguments, see `rosa create cluster --help | grep cidr`. In the interactive mode, you are prompted for the settings.
+====
+
+* To create a cluster with STS using interactive prompts:
 +
 [source,terminal]
 ----
 $ rosa create cluster --interactive
 ----
 
-. To check the status of your cluster and retrieve your cluster ID, enter the
-next command. The 'State' field changes from 'pending' to 'installing' to 'ready':
+. Create the cluster-specific Operator IAM roles. The roles created include the relevant prefix for the cluster name:
++
+[source,terminal]
+----
+$ rosa create operator-roles --cluster <cluster_name|cluster_id> <1>
+----
+<1> Replace `<cluster_name|cluster_id>` with the cluster name or the ID of the cluster.
++
+You can select from the following modes in the interactive prompt:
++
+* `auto`: The account roles and and policies are created directly using the current AWS account.
+* `manual`: The role and policy JSON files are saved in the current directory. The command output includes the `aws` CLI commands required to create the roles.
++
+[NOTE]
+====
+Alternatively, you can specify the mode by using the `--mode auto -y` or `--mode manual` CLI arguments.
+====
+
+. Create the OpenID Connect (OIDC) provider that the Operators will use to authenticate:
++
+[source,terminal]
+----
+$ rosa create oidc-provider --mode manual --cluster <cluster_name|cluster_id> <1>
+----
+<1> Replace `<cluster_name|cluster_id>` with the cluster name or the ID of the cluster.
++
+You can select from the following modes in the interactive prompt:
++
+* `auto`: The OIDC provider is created directly using the current AWS account.
+* `manual`: The OIDC provider JSON files and the thumbprint are saved in the current directory. The command output includes the `aws` CLI commands required to create the OIDC provider.
```

Comment:
```suggestion
* `manual`: The command output includes the `aws` CLI commands required to create the OIDC provider.
```
(there are no files generated by this command)

Reply:
Thank you for clarifying this. Resolved.